src/main/java/com/google/devtools/build/lib/analysis/LocationExpander.java

```diff
@@ -122,7 +122,6 @@ private LocationExpander(
    * $(execpath)/$(execpaths) using Artifact.getExecPath().
    *
    * @param ruleContext BUILD rule
-   * @param labelMap A mapping of labels to build artifacts.
```

Comment:
This looks like a leftover diff from previous versions.

Reply:
Nope, this was in the original code and was causing errors to show up since it referenced a no-longer-existing parameter. So I deleted it, and my IDE was Teh Happeez™.

Reply:
Ah, got it...